File: omnes.Web/Modules/Parametros/Etiquetas/EtiquetasForm.cs
using Serenity.ComponentModel;

namespace omnes.Parametros.Forms;

[FormScript("Parametros.Etiquetas")]
[BasedOnRow(typeof(EtiquetasRow), CheckNames = true)]
public class EtiquetasForm
{
    public string Descripcion { get; set; }
    public bool Baja { get; set; }
    public string Observaciones { get; set; }
    public int IdTennant { get; set; }
}

File: omnes.Web/Modules/ServerTypes/Parametros/EtiquetasForm.ts
import { StringEditor, BooleanEditor, IntegerEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface EtiquetasForm {
    Descripcion: StringEditor;
    Baja: BooleanEditor;
    Observaciones: StringEditor;
    IdTennant: IntegerEditor;
}

export class EtiquetasForm extends PrefixedContext {
    static formKey = 'Parametros.Etiquetas';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!EtiquetasForm.init)  {
            EtiquetasForm.init = true;

            var w0 = StringEditor;
            var w1 = BooleanEditor;
            var w2 = IntegerEditor;

            initFormType(EtiquetasForm, [
                'Descripcion', w0,
                'Baja', w1,
                'Observaciones', w0,
                'IdTennant', w2
            ]);
        }
    }
}